Getting warning CS8625 for code generated async entities

I have this simple Player class with an `AsyncEntity` attribute.
```csharp
using AltV.Net.Async.CodeGen;

public partial interface IPlayer : AltV.Net.Elements.Entities.IPlayer
{
	public ulong AccountId { get; set; }
	public bool Spawned { get; set; }
}

[AsyncEntity(typeof(IPlayer))]
public partial class Player : AltV.Net.Elements.Entities.Player, IPlayer
{
	public ulong AccountId { get; set; }
	public bool Spawned { get; set; }

	public Player(ICore server, IntPtr nativePointer, ushort id) : base(server, nativePointer, id)
	{
		AccountId = 0;
		Spawned = false;
	}
}
```
The generated code gives me the warning:
`.../AltV.Net.Async.CodeGen/AltV.Net.Async.CodeGen.AsyncEntityGenerator/GeneratedAsyncEntities.cs(35,36): warning CS8625: Cannot convert null literal to non-nullable reference type. [.../myproject.csproj]`.

I cannot see how the generated code looks like, so it's impossible to know what is causing this warning.

--- a/api/AltV.Net.Async.CodeGen/AsyncEntityGenerator.cs
+++ b/api/AltV.Net.Async.CodeGen/AsyncEntityGenerator.cs
@@ -358,7 +358,7 @@ public partial class {@class.Name}{classBaseDeclaration} {{
     }}
 
     private class Async : {asyncEntityType}<{@interface}>, {@interface} {{
-        public Async({@interface} player, AltV.Net.Async.IAsyncContext asyncContext) : base(player, asyncContext) {{ }}
+        public Async({@interface} player, AltV.Net.Async.IAsyncContext? asyncContext) : base(player, asyncContext) {{ }}
 
         public {@interface} ToAsync(AltV.Net.Async.IAsyncContext asyncContext) {{
             return asyncContext == AsyncContext ? this : new Async(BaseObject, asyncContext);

